### PR TITLE
Add code samples for alert widget and add information about modalClass option

### PR DESCRIPTION
**Pull request by serhiyzhovnir**
## Purpose of this pull request

This pull request (PR) adds code samples of the alert widget initialization.
Also, it adds information about the **modalClass** widget option.

## Affected DevDocs pages
- https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_alert.html
- https://devdocs.magento.com/guides/v2.2/javascript-dev-guide/widgets/widget_alert.html
- https://devdocs.magento.com/guides/v2.1/javascript-dev-guide/widgets/widget_alert.html

## Links to Magento source code

- https://github.com/magento/magento2/blob/2.1/app/code/Magento/Ui/view/base/web/js/modal/alert.js#L17
- https://github.com/magento/magento2/blob/2.2/app/code/Magento/Ui/view/base/web/js/modal/alert.js#L20
- https://github.com/magento/magento2/blob/2.3/app/code/Magento/Ui/view/base/web/js/modal/alert.js#L20
- https://github.com/magento/magento2/blob/2.1/app/code/Magento/Sales/view/adminhtml/web/order/create/scripts.js#L1291
- https://github.com/magento/magento2/blob/2.2/app/code/Magento/Sales/view/adminhtml/web/order/create/scripts.js#L1090
- https://github.com/magento/magento2/blob/2.3/app/code/Magento/Sales/view/adminhtml/web/order/create/scripts.js#L1473

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/javascript-dev-guide/widgets/widget_alert.md [hunk 1]
```diff
@@ -107,14 +108,83 @@ The title of the alert window.
...
+The CSS class of the alert window.
+
+**Type**: String.
+
+**Default value**: `'confirm'`
+
 ## Events {#alert_events}
 
 The alert widget implements a single event: the `always` callback.
+The `always` callback called when a modal is closed.
```
Comment (erikmarr) [inline guides/v2.1/javascript-dev-guide/widgets/widget_alert.md]:
```suggestion
The `always` callback is invoked when a modal window is closed.
```

**Comment by serhiyzhovnir** [comment]
Hi @erikmarr 
The required changes were added.
Can you, please, check it again?
Thank you!

**Comment by erikmarr** [comment]
running tests

**Comment by erikmarr** [comment]
Thanks @serhiyzhovnir!

**Pull request merged**